corehq/motech/repeaters/management/commands/find_missing_repeat_records.py

```diff
@@ -193,16 +193,19 @@ def find_missing_case_repeat_records_for_domain(domain, startdate, enddate, shou
 def find_missing_case_repeat_records_for_case(case, domain, repeaters, startdate, enddate, should_create=False):
     successful_count = missing_all_count = missing_create_count = missing_update_count = 0
 
-    repeat_records = RepeatRecord.objects.filter(domain=domain, payload_id=case.get_id).order_by()
+    repeat_records = RepeatRecord.objects.filter(
+        domain=domain,
+        payload_id=case.get_id,
+        registered_at__gte=startdate,
```

Comment:
Why not also do the `enddate` condition in here?

Reply:
Because on line 208 an iteration is done with the condition `record.registered_at.date() > enddate`. In other words, we need records on both sides of `enddate`.